static/src/stylesheets/module/facia/_container--video.scss

```diff
@@ -0,0 +1,340 @@
+$video-width-desktop: 700px;
+
+.fc-container--video {
+    background-color: darken($media-background, 7%);
+    padding-bottom: 0;
+    margin-bottom: $gs-baseline;
+
+    .gs-container {
+        background-color: $media-background;
+        border-top: 1px solid $media-default;
+    }
+
+    .fc-container__inner {
+        border-top: 0;
+
+        @include mq(leftCol) {
+            display: none;
+        }
+    }
+
+    .fc-item__image-container {
+        display: block;
+    }
+
+    .u-responsive-ratio {
+        padding-bottom: 56.3%;
+    }
+
+    .media__container--active {
+        z-index: 5;
+    }
+
+    .vjs-paused.vjs-has-started .vjs-control-bar {
+        bottom: -$gs-baseline * 5 !important;
+    }
+
+    .vjs-big-play-button {
+        @include mq(desktop) {
+            display: none;
+        }
+    }
+
+    .media__container--hidden {
+        display: block !important;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+
+        video {
+            display: none;
+        }
+
+        .gu-media-wrapper {
+            background-color: transparent;
+        }
+    }
+
+    .fc-item__video-fallback,
+    .gu-media {
+        @include mq(desktop) {
+            opacity: .3;
+            transition: opacity .4s ease-out;
+        }
+    }
+
+    @include mq($until: desktop) {
+        .u-responsive-ratio video {
+            top: inherit;
+            bottom: 0;
+            height: auto;
+        }
+
+        .fc-item__video-container .vjs-big-play-button {
+            top: (gs-height(3) - (get-line-height(textSans, 3) + 4px)) / 2;
+        }
+
+        .fc-item__video-fallback {
+            margin-top: gs-height(3) - (get-line-height(textSans, 3) + 4px);
+        }
+    }
+}
+
+.video-title {
+    color: $media-default;
+    display: block;
+    margin-bottom: $gs-baseline;
+}
+
+.video-playlist {
+    position: relative;
+    overflow: hidden;
+
+    @include mq(desktop, leftCol) {
+        background-color: darken($media-background, 5%); 
+    }
+}
+
+@for $i from 0 through 20 {
```

Comment:
I did this in css as I didn't want to do breakpoints in javascript and the value we change is used elsewhere in the sass. So if the breakpoints or the width of the video changes, you would have to change it multiple places.